.env.example

```
@@ -28,4 +22,21 @@
+NEXT_PUBLIC_POSTHOG_HOST="https://us.i.posthog.com"
+
+# AI Services & Infrastructure
+
+# Upstash Redis (for AI rate limiting and caching)
+# Get from: https://upstash.com/
+UPSTASH_REDIS_REST_URL="https://your-redis-url.upstash.io"
+UPSTASH_REDIS_REST_TOKEN="your-redis-token-here"
+
+# Weather API (for smart gardening recommendations)
+# Get from: https://www.weatherapi.com/
+WEATHER_API_KEY="your-weather-api-key-here"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)